src/components/forms/form-inputs/Toggle.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import "./Toggle.scss";
+
+const ToggleField = ({ value, setValue, className, ...props }) => {
+  // const [field, meta, helpers] = useField(props);
+  const toggleValue = () => setValue(!value);
+  let classes = "toggle form-group";
```

Comment:
Can classes be a const?

Reply:
It has to be a let so that if additional classNames are passed down as probs they are added to it (line 8)